modules/standard/Collection.chpl

```diff
@@ -0,0 +1,116 @@
+/*
+  A 'Collection' is a data structure, a container for elements that provide support
+  for insert, lookup, remove, and iteration operations.

```

Comment:
(and if it wasn't clear, we'd want to put all of your new modules in either standard/ or packages/ together)